exercises/practice/difference-of-squares/difference_of_squares_test.odin

```diff
@@ -0,0 +1,48 @@
+package difference_of_squares
+
+import "core:testing"
+
+@(test)
+test_square_of_sum_1 :: proc(t: ^testing.T) {
+	testing.expect_value(t, square_of_sum(1), 1)
+}
+
+@(test)
```

Comment:
Is it possible to skip/ignore tests? At Exercism, we have a policy of wanting students to engage in Test-Driven Development. Part of that is getting students to solve tests one by one, which many tracks implement via special ignore/skip code. E.g., in the first test is a regular, unskipped test (using `test`), but the other tests are all skipped by default (using `xtest`): https://github.com/exercism/javascript/blob/main/exercises/practice/acronym/acronym.spec.js#L5-L12

Reply:
I think the equivalent in Odin would be to just comment out the test's procedure attribute, i.e.: `// @(test)` - that will let the test runner ignore the procedure.

I suggest we move this to its own issue, since I think it's important that the contributor and maintainer decide which tests are initially disabled in the `<slug>_test.odin` file. `verify-exercises` and `run_tesh.sh` need to be adapted so that skipped exercises can be temporarily disabled. This can be a simple `sed` pipe to strip `//` from `@test` lines. 

What I'm still fuzzy on is, what is called in production when an individual Exercism user submits their new solution to an exercise? Is that single submission isolated in some way so that `verify_exercises` can be used to only run on that? Or should a script like `run-test.sh` be able to verify a solution with no tests disable?

Reply:
>  that will let the test runner ignore the procedure.

Oh but this is only for local development. The test runner must _always_ run all the tests. Which is why test runners should "unskip" tests before testing a solution.

> What I'm still fuzzy on is, what is called in production when an individual Exercism user submits their new solution to an exercise? Is that single submission isolated in some way so that verify_exercises can be used to only run on that? Or should a script like run-test.sh be able to verify a solution with no tests disabled?

The test runner is called on a directly that contains the exercise's files as found in this directory, but with any user-submitted files overwriting existing files. No script in this repo is ever called, it'll just be a `docker run` command.

Reply:
Ah, got it, thanks for clarifying!

I've skipped all but a minimal subset of test cases in the implemented exercises. The test runner now automatically "unskips" tests before the solution is tested. I've checked that this also executes correctly in the GitHub action.